value/value.go

```diff
@@ -124,7 +124,7 @@ func NewValueUsingSchema(field *schema.Field, input *structpb.Value) (Value, err
 		return nil, status.Errorf(codes.InvalidArgument, "permissible type for '%s' is string only", field.FieldName)
 	}
 
-	return nil, status.Errorf(codes.InvalidArgument, "unsupported type '%T'", input.Kind)
+	return nil, status.Errorf(codes.InvalidArgument, "unsupported type for field name '%s'", field.Name())
```

Comment:
Can it be `type %T for field name %s` ?
Meaning include both type and name.

Reply:
discussed it offline and decided to go ahead with this message.